### ISSUE TITLE
bcftools isec never ends...

### ISSUE THREAD

**Issue by PingChen-Angela** [samtools/bcftools#1284]
Hi, I want to remove known variants (dbSNP and gnomad) from my VCF file. I used the command as below. 

bcftools isec -p test  -O z -C chr9.vcf.gz dbSNP.vcf.gz gnomad.vcf.gz

It did give me output but the program seems hanging there without exit. What could be the problem??

**Comment by pd3** [comment]
Obviously, this should not happen. Is there any chance you could provide a test case to reproduce the problem?
